pygeoapi/provider/csv_.py

```diff
@@ -163,7 +193,8 @@ def get(self, identifier):
 
         :returns: dict of single GeoJSON feature
         """
-        item = self._load(identifier=identifier)
+
+        item = self.query(identifier=identifier)
```

Comment:
@francbartoli In order to do this 
![image](https://user-images.githubusercontent.com/59408650/90308533-dc292e80-defd-11ea-9148-6a5eebc4505a.png)
I had to restructure the codebase a bit.

Reply:
@francbartoli I have asked for the code to be a bit restructured so that the actual functionality of `query`  is there and not on `_load`, I find it easier and more logic this way